README.md

```diff
@@ -227,6 +227,41 @@ firestoreService.patch('regions/32/users/1', body);
 
 An error will be thrown because of id mismatch.
 
+
+## Authentications
+
+Firestore service also supports methods to authenticate users. These are not http methods, are functions to be used with the same package.
+
+### LOGIN
+
+You can login with an already created user (You can create them in the firebase console).
+
+```js
+const email = 'email@example.com'
+const password = 'xxxxxx'
+
+
+const response = await firestoreService.login(email, password);
+```
+
+The response will have all the user info.
+
+### SIGN UP
+
+You can sign up a new user with the sign up method.
+
+```js
+const email = 'email@example.com'
+const password = 'xxxxxx'
+
+
+const response = await firestoreService.signUp(email, password);
```

Comment:
There is a firebase method that allows you to update specific information of an user

Reply:
is that method something that we are planning on supporting as well?

Reply:
I think we should allow the developer to register users based in a specific user model instead of passing this arguments and after that updating the user with the rest of them

Reply:
It's a nice improvement, i'll add it to a card